cinema/serializers.py

```diff
@@ -1 +1,90 @@
-# write serializers here
+
+from rest_framework import serializers
+from cinema.models import (
```

Comment:
```suggestion
from rest_framework import serializers

from cinema.models import (
```
please fix everywhere